test/test_webhook.py

```diff
@@ -35,6 +36,59 @@ def test_it_updates_and_returns_204_if_a_profile_is_found(test_client: FlaskClie
     assert profile.name.preferred == 'Given Names Family Name'
 
 
+def test_it_does_not_cause_db_to_get_into_broken_state(test_client: FlaskClient,
```

Comment:
to the best of my knowledge is the db _connection_ getting into a state where no more transactions can be run?
Would also be helpful to paste into the docstring of this test what we know about the error, since it makes it searchable in case it happens again

Reply:
:+1: Given this is a bit of a(n unfortunately necessary) hack, it'd be useful to have a bit more context here.

Reply:
Renamed test and added link to github issue.